apps/designer/app/designer/features/style-panel/shared/get-final-value.ts

```diff
@@ -9,19 +8,12 @@ export const isNumericString = (input: string) =>
 // @todo expose which instance we inherited the value from
 export const getFinalValue = ({
   currentStyle,
-  inheritedStyle,
   property,
 }: {
   currentStyle: Style;
-  inheritedStyle: InheritedStyle;
   property: StyleProperty;
 }): StyleValue | undefined => {
   const currentValue = currentStyle[property];
-  const inheritedValue =
-    property in inheritedStyle ? inheritedStyle[property].value : undefined;
-  if (toValue(currentValue) === "inherit" && inheritedValue !== undefined) {
-    return inheritedValue;
-  }
```

Comment:
The whole code below seems like useless too, and controls dependent.
Best IMO would be or remove or set `warnOnce` the code path should not occur there.

Reply:
Will leave it to you